content/en/docs/concepts/instrumentation/libraries.md

```diff
@@ -22,6 +22,8 @@ and document hooks. Other advantages provided by native instrumentation include:
   various consumption scenarios using a wide variety of well-documented
   OpenTelemetry extensibility points.
 
+
+
```

Comment:
```suggestion
```

undo this change

Reply:
Removing both lines takes it back to the original (looks better) so let me go with that. Thank you